src/content/peerconnection/audio/js/main.js

```diff
@@ -251,18 +251,19 @@ window.setInterval(() => {
       if (report.type === 'outbound-rtp') {
         bytes = report.bytesSent;
         packets = report.packetsSent;
-        const reportId = lastResult.get(report.id);
```

Comment:
the semantics here was wrong, this is not a "reportId". Since @henbos reviewed and did not catch this he is now volunteered to explain the getStats snowball :-p